package.json

```diff
@@ -9,14 +9,15 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "next": "14.2.4",
     "react": "^18",
     "react-dom": "^18",
-    "next": "14.2.4"
+    "react-icons": "^5.2.1"
```

Comment:
Tasarımdan iconları kullanacağımız için react-icons silinmeli. Icon'lar tasarımdan kullanılmalı.